### PR TITLE
Users table updates

### PR DESCRIPTION
**Pull request by dvira**
- Make table body height scalable to fit any screen height.
- Update table columns to contain their content - no ellipsis or overflow.
- Remove sorting tooltips.
- Temp fix for select-filter's empty state size.
- Make app background color same as screens.

### CODE REVIEW

File: src/components/Screen/UsersScreen/FiltersToolbar/style.module.scss [hunk 1]
```diff
@@ -27,3 +27,8 @@
     }
   }
 }
+
+:global(.ant-select-item-empty) > div {
+  transform: scale(0.5);
+  min-height: unset;
+}
```
Comment (dr15) [inline src/components/Screen/UsersScreen/FiltersToolbar/style.module.scss:34]: Do you want this to happen always on all pages? I would put it inside `.filters-toolbar`

File: src/components/Screen/UsersScreen/index.tsx [hunk 2]
```diff
@@ -94,8 +94,9 @@ export default function UsersScreen(): ReactElement {
             data={users}
             columns={getColumns(onHeaderCellClick)}
             sortDirections={['descend', 'ascend']}
+            showSorterTooltip={false}
+            scroll={{ y: 500 }} // Just an arbitrary value that's overwritten by css
```
Comment (dr15) [inline src/components/Screen/UsersScreen/index.tsx:98]: Could you add a note why you would need this if it's being overwritten?

File: src/components/Screen/UsersScreen/FiltersToolbar/style.module.scss (hunk 1 above)
Comment (dvira) [reply src/components/Screen/UsersScreen/FiltersToolbar/style.module.scss:34]: No I don't, unfortunately I'm targeting the select's menu which is placed outside of `.filters-toolbar`, so doing what you say will not yield the desired result.
I wrote in the PR description that this is a temporary fix, and we would need to address it in the future.

File: src/components/Screen/UsersScreen/index.tsx (hunk 2 above)
Comment (dvira) [reply src/components/Screen/UsersScreen/index.tsx:98]: I thought it's pretty obvious that I'm enabling scrolling and I just couldn't leave an empty object, I guess it isn't. 
I updated the comment to reflect that.

Comment (dr15) [reply src/components/Screen/UsersScreen/index.tsx]: Thanks!

**Pull request merged**